sourcetools/com.ibm.uma/com/ibm/uma/om/Module.java

```diff
@@ -64,7 +64,12 @@ public void addArtifact(Artifact artifact) throws UMAException {
 	}
 	
 	public void addExports(String group, Exports exps) {
-		exports.put(group, exps);
+		if ( exports.keySet().contains(group) ) {
+			exps.addExports(exports.get(group));
```

Comment:
To avoid unexpected problems, and just as good practice, please create a new Exports rather than modifying the parameter.

Reply:
@pshipton Fixed as suggested, thx!